test/suite

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
We use #!/usr/bin/env sh  for shellfire scripts so that we don't force a particular shell on folks.